src/AppBundle/Entity/Post.php

```diff
@@ -80,10 +82,21 @@ class Post
      */
     private $comments;
 
+    /**
+     * @var Collection|Tag[]
```

Comment:
As far as I can tell the default is `ArrayCollection`. You should either consider it as a bad practice and do it with just `array` or what you consider better; or use the default and show how to do it right